src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -642,6 +644,67 @@ void ObjectMgr::LoadCreatureTemplateModels()
     TC_LOG_INFO("server.loading", ">> Loaded %u creature template models in %u ms", count, GetMSTimeDiffToNow(oldMSTime));
 }
 
+void ObjectMgr::LoadCreatureTemplatePersonal()
+{
+    uint32 oldMSTime = getMSTime();
+
+    //                                               0      1                 2                     3
+    QueryResult result = WorldDatabase.Query("SELECT Entry, EntryForSummoner, GroundMountDisplayID, FlightMountDisplayID FROM creature_template_personal");
+
+    if (!result)
+    {
+        TC_LOG_INFO("server.loading", ">> Loaded 0 creature template personal definitions. DB table `creature_template_personal` is empty.");
+        return;
+    }
+
+    uint32 count = 0;
+
+    do
+    {
+        Field* fields = result->Fetch();
+
+        CreaturePersonalInfo creaturePersonal = { };
+
+        creaturePersonal.Entry                = fields[0].GetUInt32();
+        creaturePersonal.EntryForSummoner     = fields[1].GetUInt32();
+        creaturePersonal.GroundMountDisplayID = fields[2].GetUInt32();
+        creaturePersonal.FlightMountDisplayID = fields[3].GetUInt32();
+
+        if (!creaturePersonal.Entry)
```

Comment:
check if the entry actually exists aka check if `CreatureTemplate const* cInfo = GetCreatureTemplate(creaturePersonal.Entry);` is null